services/ymax-planner/src/pending-tx-manager.ts

```diff
@@ -85,6 +85,9 @@ const cctpMonitor: PendingTxMonitor<CctpTx, EvmContext> = {
     const { txId, destinationAddress, amount } = tx;
     const logPrefix = `[${txId}]`;
 
+    if (destinationAddress === undefined)
+      throw Fail`CctpTxdestinationAddress is missing; got ${destinationAddress}`;
```

Comment:
In this case `destinationAddress` is always undefined` so no point in logging it